apidef/oas/middleware.go

```diff
@@ -1416,6 +1425,7 @@ func (v *VirtualEndpoint) ExtractTo(meta *apidef.VirtualMeta) {
 	}
 }
 
+// EndpointPostPlugins is a list of EndpointPostPlugins. It's used where multiple plugins can be run.
```

Comment:
i don't think we can run multiple post plugins now, it's kept as an array for future purposes I assume

Reply:
Future, tests, anyway it's reasonable too keep it around (even if it's orphaned right now)